### PR TITLE
Create MusicAlbum class and Genre class

### LINKED ISSUE #9: [1px] (Team member #2) Create MusicAlbum class
All `MusicAlbum` class properties visible in the diagram should be defined and set up in the constructor method.

### LINKED ISSUE #10: [1px] (Team member #2) Create Genre class
Create `Genre `class with an association to the Item class (in a separate .rb file).

All `Genre `class properties visible in the diagram should be defined and set up in the constructor method.

### LINKED ISSUE #11: [2pt] (Team member #2) Implement methods
- `add_item` method in the Genre class
  - should take an instance of the Item class as an input
  - should add the input item to the collection of items
  - should add self as a property of the item object (by using the correct setter from the item object)
- `can_be_archived?()` in the MusicAlbum class
   - should override the method from the parent class
   - should return true if parent's method returns true AND if on_spotify equals true
   - otherwise, it should return false

### PR DESCRIPTION
**Pull request by newhoteng**
## Create MusicAlbum class and Genre class
In this milestone I:
- Created the `MusicAlbum` class and implemented its methods
- Created the `Genre` class and implemented its methods
- Implemented setter methods in the `Items` class

### CODE REVIEW

File: music_album.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, archived: false, on_spotify: false)
+    super(publish_date, archived: archived)
+    @on_spotify = on_spotify
```
Comment (Jeddaa) [inline music_album.rb:8]: welldone @newhoteng but is the archived on line 6 different from line 7? I am asking because the values are different

**Review by Jeddaa — CHANGES_REQUESTED** [review]
Hi @newhoteng 
Almost there! There are some changes that need to be implemented in order to proceed to the next stage 👍

almost


## Highlights 👏🏻

✔️ No Linter errors
✔️ Good commit messages

## Required Changes ♻️

- Check inline comments for required changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: music_album.rb (hunk 1 above)
Comment (newhoteng) [reply music_album.rb:8]: @Jeddaa  They are the same, just like the publish date. Let's discuss it this morning so I can understand you better.

**Review by abel-tefera — APPROVED** [review]
Good work @newhoteng. PR is approved!

**Pull request merged**